### ISSUE TITLE
"day is out of range for month" (leap day)

### ISSUE THREAD

**Issue by jsha** [letsencrypt/letsencrypt#2564]
In a [Travis job](https://travis-ci.org/letsencrypt/boulder/jobs/112506704) run at 2016-02-29T03:47:20Z, I got a failure with this message:

```
Attempting to renew cert from /tmp/leitrssL/conf/renewal/le.wtf.conf produced an unexpected error: day is out of range for month. Skipping.
```

I suspect that there is an error in the day-parsing code that doesn't understand leap days (Feb 29). Note that this will make all our integration test runs fail for most of the day tomorrow.

**Comment by schoen** [comment]
Filed upstream bug at https://github.com/bear/parsedatetime/issues/155.

**Comment by schoen** [comment]
Apparently it may already have been fixed upstream, but not in the version available on PyPi.

**Comment by bmw** [comment]
While `pip` prefers the latest available version, `letsencrypt-auto` pins specific versions of our dependencies for increased stability. While unlikely to be a problem for our users, I created #2567 to make sure we pin a version of `parsedatetime` that fixes the problem in le-auto.

**Comment by bmw** [comment]
fixed in #2566
